src/telepathy/telepathy-stubs.c

```diff
@@ -303,19 +303,6 @@ void sipe_backend_notify_error(SIPE_UNUSED_PARAMETER struct sipe_core_public *si
 			       SIPE_UNUSED_PARAMETER const gchar *title,
 			       SIPE_UNUSED_PARAMETER const gchar *msg) {}
 
-/** USER *********************************************************************/
-
-void sipe_backend_user_feedback_typing(SIPE_UNUSED_PARAMETER struct sipe_core_public *sipe_public,
```

Comment:
The build fails during linking you say? Hmm, maybe telepathy plugin wrongly tries to link with both telepathy and purple backends (which each define their own versions of those functions). Can you post your full build log?